aten/src/ATen/native/mps/operations/Linear.mm

```diff
@@ -113,42 +80,49 @@ Tensor _mps_linear(
           }
           else
           {
-            MPSGraphTensor* xMulWTTensor = [mpsGraph matrixMultiplicationWithPrimaryTensor:inputTensor
+            MPSGraphTensor* inputFlattened = inputTensor;
```

Comment:
Create an issue to track removing this when OS side changes land which have a more general fix.